.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug report
+description: File a bug report
+labels: ["bug"]
+
+body:
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to reproduce
+      description: Steps to reproduce the bug.
+      value: |
+        1.
+        2.
+        3.
+        4.
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected behaviour
+      description: What should have happened.
+      placeholder: "Example: The validation should have caught the issue."
+    validations:
+      required: true
+
+  - type: textarea
+    id: current-behaviour
+    attributes:
+      label: Current behaviour
+      description: What actually happened. If you were given an error, please provide it here.
+      placeholder: "Example: I received an `APIError`."
+    validations:
+      required: true
+
+  - type: textarea
+    id: version
+    attributes:
+      label: severn version
+      description: The version of severn you're using. Obtainable though `python -m severn`.
+      placeholder: "1.0.0"
+    validations:
+      required: true
+
+  - type: textarea
+    id: system-info
+    attributes:
+      label: Extra information
+      description: The output from the Information section of `python -m severn`.
```

Comment:
Gotta make a ticket to do some UX stuff for this! Could maybe comment this out in the meantime until that gets done.